pom.xml

```diff
@@ -381,7 +381,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-checkstyle-plugin</artifactId>
-            <version>3.1.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
+            <version>3.1.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
```

Comment:
This should be updated in `external_dependencies.txt`

Reply:
This correct - https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/external_dependencies.txt#L232 ?